core/src/wallet/common/AbstractAccount.cpp

```diff
@@ -155,6 +155,10 @@ namespace ledger {
             return wallet;
         }
 
+        std::shared_ptr<api::CoreTracer> AbstractAccount::getTracer() const {
```

Comment:
can tracer be null?

Reply:
If the wallet is not correctly built yes, I added an check+exception in `WalletPool::getTracer`